resources/assets/js/features/vehicle/actionTypes.js

```diff
@@ -0,0 +1,8 @@
+export const GET_VEHICLES = 'GET_VEHICLES';
```

Comment:
rename action type to format {componentName}_{action}, for example VEHICLE_GET_ALL VEHICLE_GET VEHICLE_ADD and etc.